src/test/java/org/openstreetmap/atlas/locale/IsoCountryFuzzyMatcherTest.java

```diff
@@ -0,0 +1,30 @@
+package org.openstreetmap.atlas.locale;
+
+import java.util.stream.Collectors;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.openstreetmap.atlas.utilities.collections.Lists;
+
+/**
+ * @author lcram
+ */
+public class IsoCountryFuzzyMatcherTest
+{
+    @Test
+    public void testFuzzyMatching()
+    {
+        Assert.assertEquals("USA", IsoCountryFuzzyMatcher
+                .forDisplayCountryClosestMatch("UnitedStates").get().getIso3CountryCode());
+        Assert.assertEquals("USA", IsoCountryFuzzyMatcher
+                .forDisplayCountryClosestMatch("united States").get().getIso3CountryCode());
+        Assert.assertEquals("USA", IsoCountryFuzzyMatcher
+                .forDisplayCountryClosestMatch("united stats").get().getIso3CountryCode());
+        Assert.assertEquals("GBR", IsoCountryFuzzyMatcher
+                .forDisplayCountryClosestMatch("unitde kindgdom").get().getIso3CountryCode());
```

Comment:
fuzzy matchers are awesome!! also lol unitde  kindgom.

Reply:
They work surprisingly well

Reply:
Lol right. I've added a new command line tool that can do things like:

```
$ atlas iso-country-code USA
ISO code 'USA' matched:
US   USA   United States

$ atlas iso-country-code DMA 'kergistan'
ISO code 'DMA' matched:
DM   DMA   Dominica

Display country name 'kergistan' had no exact matches. 3 closest matches are:
KG   KGZ   Kyrgyzstan
PK   PAK   Pakistan
GE   GEO   Georgia
```

Reply:
this is a great command🙏